### PR TITLE
feat(*): add to travis

### LINKED ISSUE #12: add to CI
just as [other](https://github.com/deis/workflow) [repos](https://github.com/deis/router) do, travis.ci needs to build this codebase. since the build needs to push to dockerhub and quay.io, this issue implicitly will require a makefile refactor. see https://github.com/deis/minio/pull/9/files#r46733280 for more

### PR DESCRIPTION
**Pull request by arschles**
Fixes #12 

cc/ @mboersma @krancour

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+services:
+  - docker
+sudo: required
+before_install:
+  - wget "https://github.com/Masterminds/glide/releases/download/0.7.2/glide-0.7.2-linux-amd64.tar.gz"
+  - sudo tar -vxz -C /usr/local/bin --strip=1 -f glide-0.7.2-linux-amd64.tar.gz
+install:
+  - GLIDE_HOME=/home/travis/.glide make bootstrap
+script:
+  - make bootstrap test
```
Comment (krancour) [inline .travis.yml]: This will invoke the `bootstrap` target a second time, unnecessarily.

Comment (arschles) [inline .travis.yml]: ah yea, thanks

Comment (arschles) [inline .travis.yml]: I actually overlooked something here - I'm gonna use `quay.io/deis/go-dev` instead of installing glide here

Comment (krancour) [inline .travis.yml]: Even better!  I didn't want to force that on people, but :100: !

Comment (krancour) [inline .travis.yml]: And as I have just learned, you can change the language to `generic` then to start with no language stack.  This is largely undocumented, but @sgoings discovered it earlier today.

Comment (arschles) [inline .travis.yml]: :+1: changes coming after I finish testing all this locally

**Comment by arschles** [comment]
@krancour changes made to use quay.io/deis/go-dev

**Pull request merged**